lms/envs/common.py

```diff
@@ -2251,6 +2251,7 @@
     'openedx.features.course_search',
     'openedx.features.enterprise_support',
     'openedx.features.learner_profile',
+    'lms.djangoapps.completion',
```

Comment:
Should we actually move this into `openedx.core.djangoapps` per the README notes at https://github.com/edx/edx-platform/tree/master/openedx ?

Reply:
Maybe in the new openedx.features directory?  I think that might be a good fit for it (though it's existence is probably newer than the last time the README was updated.  @robrap might have an opinion on that.  I believe he's been involved in some work on improving the way the edx-platform code base is organized.

Reply:
For some reason when running migrations and just running the devstack LMS in #16112 I'm seeing:

> /edx/app/edxapp/edx-platform/lms/djangoapps/completion/models.py:94: RemovedInDjango19Warning: Model  class lms.djangoapps.completion.models.BlockCompletion doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.

Reply:
This may be unrelated, but John Eskew passed on the following details around setting up an AppConfig while working on his Django upgrade work. It came up in the context of importing signals.  But, the following quote can be found in these docs:
> New applications should avoid default_app_config. Instead they should require the dotted path to the appropriate AppConfig subclass to be configured explicitly in INSTALLED_APPS.

John wrote:
> Here's the commit from my branch:
> https://github.com/edx/edx-platform/commit/04d662cb396102a44e94f1fb36f2d45e04126f06
>
> and here's the Django docs:
> https://docs.djangoproject.com/en/1.11/ref/applications/

Reply:
@robrap, @bradenmacdonald: I added an AppConfig, and squashed my commits.  Assuming tests pass, this should be ready to merge, but I no longer have merge permissions on edx-platform.